support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutFormGifting.jsx

```diff
@@ -227,21 +223,26 @@ function WeeklyCheckoutFormGifting(props: PropTypes) {
           </FormSection>
           <FormSection title="Gift delivery date">
             <Rows>
-              <FieldsetWithError id="startDate" error={firstError('startDate', props.formErrors)} legend="Gift delivery date">
+              <RadioGroup
+                id="startDate"
+                error={firstError('startDate', props.formErrors)}
+                legend="Gift delivery date"
+              >
                 {days.map((day) => {
                   const [userDate, machineDate] = [formatUserDate(day), formatMachineDate(day)];
                   return (
-                    <RadioInput
-                      appearance="group"
-                      text={userDate}
+                    <Radio
+                      label={userDate}
+                      value={userDate}
                       name={machineDate}
                       checked={machineDate === props.startDate}
                       onChange={() => props.setStartDate(machineDate)}
+                      onClick={() => props.setStartDate(machineDate)}
```

Comment:
There's an `onChange` and `onClick` handler here, whilst on other `<Radio>` components we've just used `onChange`, is the `onClick` necessary?

Reply:
Yes, click isn't needed. Thanks